e2e/docker-compose.yml

```diff
@@ -13,23 +13,26 @@ services:
       - NET_ADMIN
     expose:
       - 1194
+      - 4050
+    volumes:
+      - ../../e2e/myst-provider:/var/lib/mysterium-node
     command: >
       --ipify-url=http://ipify:3000
+      --location.country=e2e-land
+      --localnet
       --broker-address=broker
       --discovery-address=http://discovery/v1
-      --localnet
-      --ether.client.rpc=http://local-node:8545
-      --location.country=e2e-land
+      --ether.client.rpc=http://geth:8545
       service
-      --openvpn.port=3000
```

Comment:
Custom openvpn port was a nice test, to make sure that port is nowhere hardcoded in our consumer/provider code. Why to remove it?

Reply:
Reverted